swaglyrics_backend/issue_maker.py

```diff
@@ -340,9 +340,9 @@ def update():
 
             if issue['status_code'] == 201:
                 print(f'Created issue on the GitHub repo for {song} by {artist}.')
-                return 'Lyrics for that song may not exist on Genius. ' \
-                       'Created issue on the GitHub repo for {song} by {artist} to investigate ' \
-                       'further. \n{link}'.format(song=song, artist=artist, link=issue['link'])
+                return f'Lyrics for that song may not exist on Genius. ' \
```

Comment:
you don't need it here, only in the strings that contain the variable

Reply:
Done